src/redux/configureStores.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
+import books from './books/books';
+import categories from './categories/categories';
+
+const root = combineReducers({ books, categories });
+
+export default root;
```

Comment:
After combining the reducers we need to create the store and export it as well.

- [ ] Kindly import `createStore` from `redux` in the first line.

- [ ] Next, you need to create the store using the `root` variable, like so:
  ```
  const store = createStore(
    root,
  );
  ```

- [ ] Kindly export the `store`, instead of `root`.